eng/pipelines/templates/variables/codeql.yml

```diff
@@ -0,0 +1,3 @@
+variables:
```

Comment:
Do we want to make this a more generic 1es settings type config? Or is this kind of separate from the 1es pt template settings.